### PR TITLE
#167711266 Improve UI and admin dashboard

### PR DESCRIPTION
**Pull request by nignanthomas**
#### What does this PR do?
Improve UI and admin dashboard.

#### Description of Task to be completed?
* A user should be redirected to his bookings once he logged in
* A user location should be gathered
* Admin shouldn't have the rights of deleting bookings

#### How should this be manually tested?
Clone the repo, and open the html pages in your browser.

#### What are the relevant pivotal tracker stories?
#167711266

### CODE REVIEW

**Comment by Krasivaya** [comment]
Well done
for me, I think that's cool!

**Comment by eliezerbasubi** [comment]
hey @nignanthomas , why are you mentioning feedback in your PR. I think it would be better if you just mention things that you have completed or changed than saying `feedback`. Because you are not doing this for Andela only, you're also doing this for other people who will be in same situation and try to find help from your repo.
By mentioning feedback, other people would think it's another feature in your project. Saying UI-improvement or update would be better.

**Comment by nignanthomas** [comment]
Thank you all for your feedback

**Comment by Kasulejoseph** [comment]
Consider @eliezerbasubi 's feedback.

File: UI/html/index.html [hunk 1]
```diff
@@ -17,7 +17,8 @@
     <div class="container">
       <div id="branding">
         <a href="index.html">
-          <img src="../images/bus-logo.png" alt="">
+          <!-- <img src="../images/bus-logo.png" alt=""> -->
+          <!-- <img src="https://img.icons8.com/clouds/75/000000/bus.png"> -->
```
Comment (Kasulejoseph) [inline UI/html/index.html]: Get rid of this comment code.

Comment (nignanthomas) [reply UI/html/index.html]: Okay
I'm working on it and I will merge now.
Thank you

**Pull request merged**